### PR TITLE
Tdr 1001 update ecr scan alert instructions

### PR DESCRIPTION
**Pull request by TomJKing**
Requires PR: https://github.com/nationalarchives/tdr-jenkins/pull/199

### CODE REVIEW

**Review by MancunianSam — APPROVED** [review]

File: manual/alerts/ecr-scans.md [hunk 1]
```diff
@@ -41,19 +41,14 @@ redeploying the image to see if it fixes the alert:
 * To rebuild a TDR service image, run the build for the master branch of that
   service's repository, e.g. [TDR Transfer Front End Test/master]
 * To rebuild the Jenkins Docker image, or one of the build node images, follow
-  the [Jenkins deployment] instructions. Add the `--pull --no-cache` flags when
-  running `docker build` to make sure you get the latest version of the base
-  image and that all of the image layers are rebuilt. This means that if the
-  Dockerfile tries to install any packages, Docker will install the latest
-  version rather than using the old version which is installed on the cached
-  image layer.
+  the [Jenkins deployment] instructions using the [TDR Build Jenkins Node Images] Jenkins job
```
Comment (suzannehamilton) [inline manual/alerts/ecr-scans.md:44]: Is there a separate job for building the main Jenkins image? This bullet point covers both the Jenkins image and the node images.

Comment (suzannehamilton) [reply manual/alerts/ecr-scans.md:44]: Just noticed this question was still open. Any thoughts?

Comment (suzannehamilton) [reply manual/alerts/ecr-scans.md:44]: @TomJKing Any thoughts on this one? It's been open for a while.

Comment (TomJKing) [reply manual/alerts/ecr-scans.md:44]: Sorry, thought I'd replied to this, but apparently not, as the comment was stuck in `pending` : /

There is a single job that builds both the main and node images.

**Pull request merged**